### DISCUSSION TITLE
Custom social card icon?

### DISCUSSION THREAD

**Discussion by A-Baji** [squidfunk/mkdocs-material#4128] [Q&A] [answered]
I know the docs state that only icons included by the theme are usable for social cards, but is there any workaround if I wanted to use a custom icon?

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking! Social cards are integrated with Material for MkDocs icon system, which means that custom icons are supported through [additional icons](https://squidfunk.github.io/mkdocs-material/setup/changing-the-logo-and-icons/#additional-icons). If you can use your custom icons inside your documentation, they should also work for social cards. If for some reason that doesn't work, please open an issue with a minimal reproducible example.

**Reply by A-Baji** [reply]
Thanks, I somehow missed this. Is there a specific format the icons need to be in order for them to have the correct coloring when hovered?
![image](https://user-images.githubusercontent.com/61991709/179602286-4c616bdf-de83-49cc-81c7-1918499e4487.png)

**Reply by squidfunk** [reply]
Only SVG is supported, [as mentioned in the documentation](https://squidfunk.github.io/mkdocs-material/setup/changing-the-logo-and-icons/#additional-icons).

**Reply by squidfunk** [reply]
Okay, so a huge misunderstanding: you wrote [social cards](https://squidfunk.github.io/mkdocs-material/setup/setting-up-social-cards/), but you mean [social links](https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-footer/#social-links), right? You should make sure that the SVG source does not contain any `fill` colors, or Material for MkDocs can't set the right color for the icon. As an example, [see the source of this icon](https://github.com/squidfunk/mkdocs-material/blame/master/src/.icons/logo.svg).

**Reply by A-Baji** [reply]
Oh, my bad :sweat_smile: . Thanks for the answer though. I got it working.
